src/user/controller/by-href.ts

```diff
@@ -21,12 +22,16 @@ class UserByHrefController extends Controller {
     if (hasControl && user.type === 'user') {
       hasPassword = await userService.hasPassword(user);
     }
+    if (privilegeService.hasPrivilege(user, 'admin', '*')) {
```

Comment:
Check out line 19 for a good way to call this. This returns a promise, and because there is a database call it would be better if we only run this function once per request.